packages/sanity/src/core/releases/tool/detail/documentTable/DocumentActions.tsx

```diff
@@ -22,11 +21,13 @@ export const DocumentActions = memo(
     const [discardStatus, setDiscardStatus] = useState<'idle' | 'discarding' | 'error'>('idle')
     const client = useClient(DEFAULT_STUDIO_CLIENT_OPTIONS)
     const toast = useToast()
-    const {t} = useTranslation(releasesLocaleNamespace)
+    const {t} = useTranslation()
```

Comment:
Hmm, I see now we have some labels from the global namespace (the ones you updated eg. 'bundle.action.discard-version.success'), but we have others still from releases eg. discard-version-dialog.title.

We might need to define 2 translation functions:

```
const {t: studioT} = useTranslation()
const {t: releasesT} = useTranslation(releasesLocaleNamespace)
```

Or it looks like it might be possible to combine these:
```
const {t} = useTranslations([releasesLocaleNamespace])
```

Reply:
Ignore the second way, although `useTranslations` accepts an array, it seems that only the first element gets used

Reply:
Great find, I have some other places where I do this as well but I forgot to do it here.
Thank you for catching it!